### ISSUE TITLE
Add some functions to the "Search in results" panel

### ISSUE THREAD

**Issue by pidgeon777** [pnedev/nppgtags#16]
It would be extremely useful to transform the search result window so that it could behave like this:

**Automatically filtering of search results when typing the search expression**

Example:

Let's consider a search result of this kind:

```
Find Symbol "CLR" (literal, match case) in "C:\Encoder\"
	CRC_Check.vhd
		line 131:	CLR		: in std_logic;
		line 201:	CLR		=> IMAGELINESTART,
	CRC_Encoder.vhd
		line 42:	CLR		: in std_logic;
		line 63:	if (CLR = '1') then
```

If I type the character "=" in the filter box, then the results would become:

```
Find Symbol "CLR" (literal, match case) in "C:\Encoder\"
	FCA_Check.vhd
		line 201:	CLR		=> FCALINESTART,
	FCA_Encoder.vhd
		line 63:	if (CLR = '1') then
```

If I also add a ">" to the filter box appending it to the already typed "=", it would automatically become:

```
Find Symbol "CLR" (literal, match case) in "C:\Encoder\"
	CRC_Check.vhd
		line 201:	CLR		=> IMAGELINESTART,
```

So this kind of filtering would be useful for example to look for special code statements (for example, ports, signal assignments, arrays, defines...).

Moreover, an option could be added to save some user customizable filters which could be used between sessions (for example by selecting them through a listbox).

Finally, it would be great to make the selected filter persistent so that at every search, the results are automatically filtered, if chosen to do so.

This would make NppGTags code browsing even more powerful, in my opinion.

**Comment by pnedev** [comment]
Good idea but I don't know when I'll have time to work on NppGTags again.

Meanwhile you can do searches in the results panel - while your focus is there press _CTRL + F_ and use the search box.

Thanks.

**Comment by pidgeon777** [comment]
If you will find the time please consider this suggestion of mine, it would be a really appreciated feature.
